### DISCUSSION TITLE
In series list, show books not in your library

### DISCUSSION THREAD

**Discussion by KodeStar** [joonaspaakko/audible-library-extractor#95] [Ideas]
On the overview it will say something like 3 of 6, then when you click on it it shows the books you have. It would be nice if it also showed the books you don't have (maybe with 50% opacity of something, to differentiate) so it's easy to see at a glance if it's a new book you are missing and/or to make it extra clear what that of is referring to.

For example, A Thousand Li by Tao Wong:
<img width="640" alt="image" src="https://user-images.githubusercontent.com/5698566/169490232-22644061-e16e-40a1-a551-3ae22a4cfc4b.png">

<img width="819" alt="image" src="https://user-images.githubusercontent.com/5698566/169491033-b37a0b25-ae75-4714-9f9c-5bff19ee0abd.png">

I know Travis Baldree is currently recording book 7, but it doesn't show up on Audible yet (nor for preorder), So I'm not sure what the 7 is referring to, it could be the short stories that I don't have, but there are 2 of those so I'd have thought it would be 6 of 8 in that case, or maybe wherever the data is coming from is showing that book 7 is incoming it's just not showing up on the site yet.
<img width="287" alt="image" src="https://user-images.githubusercontent.com/5698566/169490844-c66662af-453c-4258-bb59-2531adffc477.png">

**Comment by Britastic365** [comment]
I would love this as well.
